src/seedu/addressbook/AddressBook.java

```diff
@@ -431,6 +434,20 @@ private static String executeAddPerson(String commandArgs) {
         addPersonToAddressBook(personToAdd);
         return getMessageForSuccessfulAddPerson(personToAdd);
     }
+	
+	private static boolean executeAddPersonAfterEdit(String commandArgs, int index) {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.